Orders/Kyokuya/co06033202.csv

```diff
@@ -1,3 +1,3 @@
-Title#0,"""Desesperacion, mi extraña sombra"""
-Explanation#0,"""...Oh, eres tú otra vez. ¿Quieres descender tanto a la oscuridad?<br>Jejeje...luego muéstrame tu resolución. Desecha cualquier pensamiento tonto de justicia... ¿Estás dispuesto a hacer lo que sea necesario?<br>seré yo quien determine que...<br><br><yellow>※Solo Phantom<br>Objetivo: Enemigos Nv. 41+<c>"""
-Reward#0,"""+5 Puntos de Skill Phantom"""
+Title#0,"""Desesperacion, mi sombra"""
+Explanation#0,"""...Oh, eres tú otra vez.<br>¿Tanto deseas descender a la oscuridad?<br>Jejeje...luego muéstrame tu resolución.<br>Desecha cualquier pensamiento<br>ridiculo de justicia...<br>¿Estás dispuesto a hacer lo que sea necesario?<br>seré yo quien lo determine...<br><br><yellow>※Solo Phantom<br>Objetivo: Enemigos Nv. 41+<c>"""
+Reward#0,"""+5 Puntos de Skill Phantom"""
```

Comment:
Why is the EOL missing on the Kyokuya COs?

Reply:
Accidental delete fixing a conflict